generate/templates/partials/convert_from_v8.cc

```diff
@@ -3,10 +3,12 @@
   {{ cType }} from_{{ name }};
   {%if isOptional | or isBoolean %}
 
-    {%if cppClassName != 'GitStrarray'%}
-  if (args[{{ jsArg }}]->Is{{ cppClassName|cppToV8 }}()) {
+    {%if cppClassName == 'GitStrarray'%}
+    {% elsif cppClassName == 'GitBuf' %}
```

Comment:
Add a Combyne comment (i.e. `{%-- my comment --%}`) to be clear that these aren't supposed to do anything. The logic looks really weird.